### DISCUSSION TITLE
Material admonitions

### DISCUSSION THREAD

**Discussion by madeuce463** [squidfunk/mkdocs-material#4846] [Q&A]
I have added the fontawesome admonitions to my mkdocs.yaml file and these are appearing as expected in my content page. <br />
I would like to use some of the material admonitions shown in the expanded menu (+) under the code example under [Admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/). E.g., material/key-alert.<br />How should these be shown in yaml, and then how do I set these up in the content pages?<br /> I've tried a few things that seem to follow the same pattern as the fontawesome, but nothing is working. Thanks!

**Comment by squidfunk** [comment]
I don't understand the question, I'm sorry. I think you're mixing icons and admonitions somehow.

> but nothing is working

Without knowing what you tried exactly, given a reproduction, we're unable to help you.

**Reply by madeuce463** [reply]
Yes, I mean the icons that are shown with the admonitions - see below/attached. Thanks!

![Capture_Admonition-Icons](https://user-images.githubusercontent.com/117672163/211810084-bc4579f6-11fc-4fc7-89b7-c69d0428b241.png)

**Reply by josh-wong** [reply]
@madeuce463 I've tried what you described by using both icons from Material Design and Simple Icons. As shown in [Admonition icons](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#admonition-icons), it seems that only the icons for Octicon and FontAwesome can be used in admonitions in `mkdocs.yml`.

I have confirmed, however, that Material Design icons can be used in admonitions by adding them to `docs/stylesheets/extras.css`. For details, please see [Custom admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#custom-admonitions).

**Reply by madeuce463** [reply]
@josh-wong Thanks so much for your response - this worked. Thanks for your help!
I deleted a post with further questions, but what you provided did work.
